lte/gateway/c/session_manager/test/BUILD.bazel

```diff
@@ -176,6 +177,10 @@ cc_test(
     size = "small",
     srcs = ["test_sessiond_integ.cpp"],
     flaky = True,
+    # This test unfortunately brings up a test server that collides with another test (cloud_reporter)
+    # Marking it as exclusive so that the test is not run in parallel with other tests
+    # See https://docs.bazel.build/versions/4.2.1/test-encyclopedia.html#tag-conventions
+    tags = ["exclusive"],
```

Comment:
In golang to avoid that we set the port to 0. That forces golang to choose any available port.

I guess cpp does not have that feature, but maybe we could try to come up with something like that in the future.

Reply:
yeah I tried to look into this at some point but didn't find anything too easy unfortunately. :(